templates/broker.servicecatalog.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: servicecatalog.k8s.io/v1beta1
 kind: {{ broker_kind }}
 metadata:
-  name: {{ broker_name }}
+  name: "{{ cluster_server_broker_resource_name }}"
```

Comment:
you'll want to set this in `defaults/main.yml` since the apb.yml isn't really used when launching the pod.

Reply:
@djzager good call. I've added it there.

Reply:
you can make the route's name `broker` and it will end up being `broker-{{ broker-namespace }}.{{ routing_suffix }}` I think that is cleaner in the end.